geopandas/geodataframe.py

```diff
@@ -636,6 +636,37 @@ def explode(self):
         return geo_df
 
 
+def points_from_xy(x, y, z=None):
+    """
+    Generate list of shapely.Point geometries from x, y(, z) cordinates.
```

Comment:
```suggestion
    Generate list of shapely Point geometries from x, y(, z) coordinates.
```

(since otherwise it looks like an import which is not strictly true)